tests/unit/test_bigquery.py

```diff
@@ -0,0 +1,229 @@
+"""'test_bigquery.py'."""
+
+from google.oauth2 import service_account
+import numpy as np
+import pandas as pd
+import pytest
+
+from viadot.exceptions import APIError, CredentialError
+from viadot.sources.bigquery import BigQueryCredentials
+from viadot.utils import skip_test_on_missing_extra
+
+
+try:
+    from viadot.sources import BigQuery
+except ImportError:
+    skip_test_on_missing_extra(source_name="BigQuery", extra="bigquery")
```

Comment:
Shouldn't be here as the source doesn't use optional dependencies.

Reply:
[6d00a97](https://github.com/dyvenia/viadot/pull/1035/commits/6d00a97a1f690365491fb1886e6b2fabcce79f2d)